Formula/mongodb-community@3.4.rb

```diff
@@ -1,81 +0,0 @@
-class MongodbCommunityAT34 < Formula
```

Comment:
Is the rationale for removing this that 3.4 is EOL? If so, should we remove 3.6 as well?

Reply:
And, why aren't we removing 3.2 as well, if we are doing this? Do we actually need to fold in these removals to the 5.0 GA change? If not, I'd consider backing them out.